octobot_commons/databases/adaptors/tinydb_adaptor.py

```diff
@@ -15,26 +15,32 @@
 #  License along with this library.
 try:
     import tinydb
+    import tinydb.storages
+    import tinydb.middlewares
 except ImportError:
     pass
 
 import octobot_commons.databases.adaptors.abstract_database_adaptor as abstract_database_adaptor
 
 
 class TinyDBAdaptor(abstract_database_adaptor.AbstractDatabaseAdaptor):
+    WRITE_CACHE_SIZE = 5000
+
     """
     TinyDBAdaptor is an AbstractDatabaseAdaptor implemented using tinydb: a minimal python only
     local document database.
-    Warning: loads the whole file in RAM
+    Warning: loads the whole file in RAM and must be closed to ensure writing
     """
 
     def __init__(self, file_path: str):
         """
-        TinyDBAdaptor constructor
+        TinyDBAdaptor constructor.
         :param file_path: path to the database file
         """
         super().__init__(file_path)
-        self.database = tinydb.TinyDB(file_path)
+        tinydb.middlewares.CachingMiddleware.WRITE_CACHE_SIZE = self.WRITE_CACHE_SIZE
```

Comment:
Is it the only way to do change this setting?

Reply:
Right ! I just changed this